nucypher/blockchain/eth/sol/source/contracts/proxy/Dispatcher.sol

```diff
@@ -81,9 +85,9 @@ contract Dispatcher is Upgradeable {
     * @dev Fallback function send all requests to the target contract.
     * If contract not exists then result will be unpredictable (see DELEGATECALL)
     **/
-    function () public payable {
-        assert(target != 0x0);
-        bool callSuccess = target.delegatecall(msg.data);
+    function () external payable {
+        assert(target != address(0));
+        (bool callSuccess,) = target.delegatecall(msg.data);
```

Comment:
The addition of these parentheses and comma are foreign to me.  What do we have happening here?  `delegatecall` started returning a tuple?  When previously a returned a single object?

Reply:
yep, you're right
second value `bytes memory data`

Reply:
I don't recall every having seen a tuple with an anonymous member to the left of an assignment operator.  Is this typical in solidity 0.5+?

Reply:
It was added in some 0.4.2? version